src/utils/Api.js

```diff
@@ -0,0 +1,93 @@
+export default class Api {
+  constructor({ headers, baseUrl }) {
+    this._defaultHeaders = headers;
+    this._baseUrl = baseUrl;
+  }
+
+  getInitialCards() {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/cards", { headers: this._defaultHeaders })
+    );
+  }
+
+  getUserInfo() {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/users/me", { headers: this._defaultHeaders })
+    );
+  }
+
+  updateUserInfo(userInfo) {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/users/me", {
+        method: "PATCH",
+        headers: this._defaultHeaders,
+        body: JSON.stringify(userInfo),
+      })
+    );
+  }
+
+  createCard(createCardInfo) {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/cards", {
+        method: "POST",
+        headers: this._defaultHeaders,
+        body: JSON.stringify(createCardInfo),
+      })
+    );
+  }
+
+  deleteCard(cardId) {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/cards/" + cardId, {
+        method: "DELETE",
+        headers: this._defaultHeaders,
+      })
+    );
+  }
+
+  likeCard(cardId) {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/cards/" + cardId + "/likes", {
+        method: "PUT",
+        headers: this._defaultHeaders,
+      })
+    );
+  }
+
+  unlikeCard(cardId) {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/cards/" + cardId + "/likes", {
+        method: "DELETE",
+        headers: this._defaultHeaders,
+      })
+    );
+  }
+
+  updateAvatar(avatar) {
+    return this.handleresponse(
+      fetch(this._baseUrl + "/users/me/avatar", {
+        method: "PATCH",
+        headers: this._defaultHeaders,
+        body: JSON.stringify(avatar),
+      })
+    );
+  }
+
+  handleresponse(request) {
```

Comment:
`handleresponse`

Every variable in the project should be in `camelCase` 





`handleResponse`